changes-set.txt

```diff
@@ -27,14 +27,36 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+ 4-Oct-22 brresi    brresi2
+ 4-Oct-22 opelresi  opelidres
+ 4-Oct-22 brresg    brresgOLD2
+ 4-Oct-22 opelresg  opelresgOLD2
+ 4-Oct-22 brres     brresOLD2
+ 4-Oct-22 brres     brresi
+ 4-Oct-22 opelres   opelresOLD2
+ 4-Oct-22 opelres   opelresi
+ 4-Oct-22 brresALTV brres       moved from PM's mathbox to main set.mm
+ 4-Oct-22 opelresALTV opelres   moved from PM's mathbox to main set.mm
+ 4-Oct-22 inxpssres [same]      moved from PM's mathbox to main set.mm
+ 4-Oct-22 biancomd  [same]      moved from PM's mathbox to main set.mm
+ 4-Oct-22 biancomi  biancom     moved from PM's mathbox to main set.mm
```

Comment:
anbi1ci and anbi1cd are also moved from PM's mathbox to main set.mm!?

Reply:
done ( https://github.com/metamath/set.mm/pull/2873/commits/1961e2a3ae7faae6628f84557a3e4cf029cdc556 )